packages/dashboard/src/components/tasks/task-page.tsx

```diff
@@ -43,7 +43,7 @@ export function TaskPage() {
         undefined,
         11,
         page * 10,
-        '-unix_millis_start_time',
```

Comment:
Why is this changed?

Reply:
https://osrf.slack.com/archives/C01ECP3GBTQ/p1641831025038800

Reply:
Yeah, `unix_millis_start_time` should be working now. Having it undefined means unspecified sorting.

Reply:
Cool, didn't know it was fixed already, reverting it then.